dev-tools/reconfigurator-cli/tests/output/cmd-example-stdout

```diff
@@ -128,20 +129,21 @@ parent:    02697f74-b14a-4418-90f0-c28b2a3a6aa9
     ---------------------------------------------------------------------------------------------
-    crucible          09937ebb-bb6a-495b-bc97-b58076b70a78   in service    fd00:1122:3344:103::2c
-    crucible          a999e5fa-3edc-4dac-919a-d7b554cdae58   in service    fd00:1122:3344:103::26
-    crucible          b416f299-c23c-46c8-9820-be2b66ffea0a   in service    fd00:1122:3344:103::27
-    crucible          b5d5491d-b3aa-4727-8b55-f66e0581ea4f   in service    fd00:1122:3344:103::2b
-    crucible          cc1dc86d-bd6f-4929-aa4a-9619012e9393   in service    fd00:1122:3344:103::24
-    crucible          cd3bb540-e605-465f-8c62-177ac482d850   in service    fd00:1122:3344:103::29
-    crucible          e8971ab3-fb7d-4ad8-aae3-7f2fe87c51f3   in service    fd00:1122:3344:103::25
-    crucible          f3628f0a-2301-4fc8-bcbf-961199771731   in service    fd00:1122:3344:103::2d
-    crucible          f52aa245-7e1b-46c0-8a31-e09725f02caf   in service    fd00:1122:3344:103::2a
-    crucible          fae49024-6cec-444d-a6c4-83658ab015a4   in service    fd00:1122:3344:103::28
-    crucible_pantry   728db429-8621-4e1e-9915-282aadfa27d1   in service    fd00:1122:3344:103::23
-    internal_dns      e7dd3e98-7fe7-4827-be7f-395ff9a5f542   in service    fd00:1122:3344:2::1   
-    internal_ntp      4f2eb088-7d28-4c4e-a27c-746400ec65ba   in service    fd00:1122:3344:103::21
-    nexus             c8aa84a5-a802-46c9-adcd-d61e9c8393c9   in service    fd00:1122:3344:103::22
+    crucible          09937ebb-bb6a-495b-bc97-b58076b70a78   in service    fd00:1122:3344:103::2b
+    crucible          a999e5fa-3edc-4dac-919a-d7b554cdae58   in service    fd00:1122:3344:103::25
+    crucible          b416f299-c23c-46c8-9820-be2b66ffea0a   in service    fd00:1122:3344:103::26
+    crucible          b43ce109-90d6-46f9-9df0-8c68bfe6d4a0   in service    fd00:1122:3344:103::2e
+    crucible          b5d5491d-b3aa-4727-8b55-f66e0581ea4f   in service    fd00:1122:3344:103::2a
+    crucible          cbe91cdc-cbb6-4760-aece-6ce08b67e85a   in service    fd00:1122:3344:103::2d
+    crucible          cd3bb540-e605-465f-8c62-177ac482d850   in service    fd00:1122:3344:103::28
+    crucible          f3628f0a-2301-4fc8-bcbf-961199771731   in service    fd00:1122:3344:103::2c
+    crucible          f52aa245-7e1b-46c0-8a31-e09725f02caf   in service    fd00:1122:3344:103::29
+    crucible          fae49024-6cec-444d-a6c4-83658ab015a4   in service    fd00:1122:3344:103::27
+    crucible_pantry   e8971ab3-fb7d-4ad8-aae3-7f2fe87c51f3   in service    fd00:1122:3344:103::24
+    external_dns      cc1dc86d-bd6f-4929-aa4a-9619012e9393   in service    fd00:1122:3344:103::23
+    internal_dns      728db429-8621-4e1e-9915-282aadfa27d1   in service    fd00:1122:3344:2::1   
+    internal_ntp      c8aa84a5-a802-46c9-adcd-d61e9c8393c9   in service    fd00:1122:3344:103::21
+    nexus             e7dd3e98-7fe7-4827-be7f-395ff9a5f542   in service    fd00:1122:3344:103::22
```

Comment:
I'm not really clear why this block and the next one changed.  It looks like the IPs shifted around?  Is that just because of the new external DNS zone too?

Reply:
I think it's more that the UUIDs _continued_ shifting around. The zone UUIDs are all coming out of a deterministic RNG, but that's for the entire blueprint. So the initial sled got a new zone (external DNS), which means it now grabbed one more zone UUID than it did previously. Going up, the "new" UUID it got went to one of the crucible zones (`4f2eb088-7d28-4c4e-a27c-746400ec65ba`) - that UUID used to belong to _this_ sled's internal NTP zone (the first one `ExampleSystemBuilder` allocates for a sled). Instead, the UUIDs for every zone on this sled (and all subsequent sleds) shifted.

Reply:
Hmm, part of the goal of RNG trees was to minimize this shifting. Can it be restructured to achieve that?

Reply:
Looking at the output, I think each sled should definitely get a separate RNG.

Reply:
Great call - after pulling in #7321, all three blocks have just a few lines shifted around: https://github.com/oxidecomputer/omicron/pull/7307/files#diff-18cea7c50c6bb09d57e37c8bf51e112a953dc2827d2fe4f300f9b6aac2f5b0fb